Gemfile

```diff
@@ -3,9 +3,11 @@ source "http://rubygems.org"
 # Specify your gem's dependencies in mongoid_orderable.gemspec
 gemspec
 
-case version = ENV['MONGOID_VERSION'] || "~> 3.1"
+case version = ENV['MONGOID_VERSION'] || "~> 4.0"
+when /5/
+  gem "mongoid", "~> 5.0.0"
```

Comment:
Just used the recommended value from the mongoid 5 docs - https://docs.mongodb.org/ecosystem/tutorial/ruby-mongoid-tutorial/ - but aye, to be consistent with your prior versions perhaps should have put ~> 5.0